docs/faq.rst

```diff
@@ -702,6 +702,29 @@ Now Borg will be bandwidth limited. Nice thing about pv is that you can change r
 .. _pipeviewer: http://www.ivarch.com/programs/pv.shtml
 
 
+How can I restore from a BTRFS snapshot to the original path?
+-------------------------------------------------------------
+
+When I do a backup of a BTRFS file system I first do a readonly snapshot and
+then back it up:
+
+::
+
+    btrfs subvolume snapshot -r /home /.snapshots/home
+    cd /.snapshots
+    borg create /path/to/repo::home_backup
+    btrfs subvolume delete /.snapshots/home
```

Comment:
This would be a bit more generic and not limited to btrfs users:

mount --bind / /rootfs
cd /rootfs
borg create /path/to/repo::root_backup .

Reply:
Note that `.` at the end.

Reply:
OK

Reply:
> Note that `.` at the end.

Why is this? Can't it be ommitted?

Reply:
borg create needs at least 1 PATH argument.

Reply:
Good to know, I will have to fix my pull mode PR then, too.